test/ctia/stores/es/crud_test.clj

```diff
@@ -568,3 +594,151 @@
               :matched high-t1-title1
               :ident ident-1
               :deleted? true}))))))
+
+(deftest docs-with-indices-test
+  (es-helpers/for-each-es-version
+      "get-docs-with-indices (and variants) shall properly return documents for given ids"
+      [5 7]
+    #(es-index/delete! % "ctia_*")
+    (helpers/fixture-ctia-with-app
+     (fn [app]
+       (let [conn-state (get-conn-state app :sighting)
+             base-sighting {:tlp "green"
+                            :groups ["group1"]}
+             total 200
+             sighting-ids (map #(str "sighting-"  %) (range total))
+             mk-title #(str "title "%)
+             sightings (map #(assoc base-sighting
+                                    :id %
+                                    :title (mk-title %))
+                            sighting-ids)
+             ;; drop some docs to check that we only match requeted ids
+             tested-sample (drop (/ total 10)
+                                 (shuffle sightings))
```

Comment:
Using `shuffle` in tests seems delicate since tests would ideally would always be deterministic/reproducible.

Is there some alternative?